<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>582</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>411</width>
    <height>582</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1169</width>
    <height>717</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icon_calculator.png</normaloff>icons/icon_calculator.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 23, 23);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="2">
     <widget class="QPushButton" name="dot">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>,</string>
      </property>
      <property name="shortcut">
       <string>,</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="num_8">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="shortcut">
       <string>8</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="num_7">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="shortcut">
       <string>7</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="factorial">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>!</string>
      </property>
      <property name="shortcut">
       <string>!</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="num_4">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="shortcut">
       <string>4</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="mul">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <property name="shortcut">
       <string>*</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: none;
  color:  rgb(211, 215, 207); 
}

</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" rowspan="3">
     <widget class="QPushButton" name="sum">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="shortcut">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="modulo">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
      <property name="shortcut">
       <string>%</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sqrt">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
      <property name="shortcut">
       <string>Shift+S</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="num_0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="shortcut">
       <string>0</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="ce">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>CE</string>
      </property>
      <property name="shortcut">
       <string>Backspace</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="num_9">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="shortcut">
       <string>9</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="num_1">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="shortcut">
       <string>1</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="add">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="shortcut">
       <string>+</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pow">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>pow</string>
      </property>
      <property name="shortcut">
       <string>^</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="div">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
      <property name="shortcut">
       <string>/</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="num_6">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="shortcut">
       <string>6</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="num_5">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="shortcut">
       <string>5</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="num_3">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="shortcut">
       <string>3</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="sub">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="shortcut">
       <string>-</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="num_2">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(25, 23, 23);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2e2a2a, stop: 1 #423c3c);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="shortcut">
       <string>2</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">num_group</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="ac">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 35px;
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(105,105,105), stop: 1 rgb(135,135,135));
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Backspace</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operator_group</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="help">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:  rgb(64, 64, 64);
  color:  rgb(211, 215, 207); 
	border-radius: 4.5px;
	
}

QPushButton:pressed {
	  background-color:  rgb(25, 23, 23); 
}</string>
      </property>
      <property name="text">
       <string>help</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="num_group"/>
  <buttongroup name="operator_group"/>
 </buttongroups>
</ui>
